manage/roles/agent-permissions.rst

```diff
@@ -53,7 +53,7 @@ Agent Permissions
                                Zammad supports `granular permissions on knowledge base categories`_.
 
                                This function allows *agents with editor* permissions to restrict
-                               roles to specific internal categories & answers.
+                               roles to specific internal categories & answers to chosen roles.
```

Comment:
Drop the "roles to" at the beginning. As now it is included twice.

Reply:
🙈